README.md

```diff
@@ -422,12 +422,47 @@ with at least one node that has persistent memory device(s). For development or
 testing, it is also possible to use a cluster that runs on QEMU virtual
 machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
 
+- **Make sure that alpha feature gates CSINodeInfo, CSIDriverRegistry are enabled**
+
+The method to configure alpha feature gates may vary, depending on the kubernetes deployment.
+
 - **Label the cluster nodes that provide persistent memory device(s)**
 
 ```sh
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Verify that the node labels have been configured correctly**
+
+```sh
+    $ kubectl get nodes --show-labels
+```
+
+Output of above command should indicate that every non-master node has labels:
+```
+pmem-csi.intel.com/node=<NODE-NAME>,storage=pmem
+```
+
+- **Install addon storage CRDs if using Kubernetes 1.13**
+
+If you are not using the test cluster described in
+[Starting and stopping a test cluster](#starting-and-stopping-a-test-cluster)
+where CRDs are installed automatically, you must install those manually:
+
+```sh
+   $ kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.13/cluster/addons/storage-crds/csidriver.yaml
+   $ kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.13/cluster/addons/storage-crds/csinodeinfo.yaml
+```
+
+- **Similar CRDs install step is needed if using Kubernetes 1.14**
```

Comment:
There's no need for a separate section.

Reply:
I created separate section only because the path has version value in it. Should we keep 1.13 path, or change to template <version> ? Every time we use template, we add inconvenience that command can not be directly copied from README

Reply:
Use the same 1.13 path for both 1.13 and 1.14. The CRD definition is frozen, there's no need to pull it from different places.